dev/bots/test.dart

```diff
@@ -1767,6 +1771,8 @@ Future<void> runFlutterWebTest(
       'FLUTTER_WEB': 'true',
     },
   );
+  // Delete metrics file.
+  metricFile.deleteSync();
```

Comment:
why do we delete this?

Reply:
This is a transitional file that contains a portion of the final test results. The content of this file will be parsed and aggregated into a file called test_results.json

Reply:
Maybe leave a TODO here and elsewhere for that to give context for future readers until that is done?

Reply:
Added a TODO and context.